### ISSUE TITLE
Doesn't show code

### ISSUE THREAD

**Issue by Sjoerd** [vue-formation/vue-prismjs#1]
Hi there,
If the code loads async, it doesn't show up in prism.
(got the code from my API using axios)
https://gyazo.com/0575588c84a4ef00ee739970bda74511 << This is what is see when I check the generated html.

**Comment by bhoriuchi** [comment]
Have you tried putting the code in a computer property and passing that to he component

**Comment by Sjoerd** [comment]
Yes

**Comment by bhoriuchi** [comment]
i added a watcher to the `code` prop and published `v0.1.1` to npm. Let me know if that fixes your issue.

Test code was

**HTML**
```html
<prism language="javascript" :code="example"></prism>
```

**JS**
```javascript
  import Prism from 'vue-prismjs'

  export default {
    mounted () {
      setTimeout(() => {
        this.example1 = 'const foo = "bar"'
      }, 2000)
    },
    components: {
      Prism
    },
    computed: {
      example () {
        return this.example1
      }
    },
    data () {
      return {
        example1: ''
      }
    }
  }
```
